webapp/graphite/finders/standard.py

```diff
@@ -68,16 +68,24 @@ def _find_paths(self, current_dir, patterns):
     match the corresponding pattern in patterns"""
     pattern = patterns[0]
     patterns = patterns[1:]
-    try:
-      entries = os.listdir(current_dir)
-    except OSError as e:
-      log.exception(e)
-      entries = []
 
+    has_wildcard = pattern.find('{') > -1 or pattern.find('[') > -1 or pattern.find('*') > -1
+
+    if has_wildcard:
+      try:
+        listdir = os.listdir(current_dir)
+      except OSError as e:
+        log.exception(e)
+        listdir = []
+    else:
+      listdir = []
+
+    entries = listdir if has_wildcard else [ pattern ]
```

Comment:
Instead of `if has_wildcard` here, why not just set `listdir = [ pattern ]` on 81?